crates/red_knot_python_semantic/resources/mdtest/exception/basic.md

```diff
@@ -41,7 +41,12 @@ except EXCEPTIONS as f:
 ## Dynamic exception types
 
 ```py
-def foo(x: type[AttributeError], y: tuple[type[OSError], type[RuntimeError]], z: tuple[type[BaseException], ...]):
+# TODO: we should not emit those errors `call-potentially-unbound-method` errors
```

Comment:
Okay, I changed the TODO message, but I'm not sure how to handle this case for now? Should we add a special case that prevents this diagnostic for `tuple.__class_getitem__`?

Or leave the TODO because those errors go away when we understand `sys.version_info` branches here?

https://github.com/python/typeshed/blob/355abb197eeabc406c3d03b65ff23245e6a38b4e/stdlib/builtins.pyi#L993-L994

Reply:
I think it's fine to leave it alone for now and let it be resolved by `sys.version_info` checking.

The real question is whether we want a diagnostic in general in such cases in user code (that can't be resolved to known-False or known-True) -- but I expect such cases are rare, so we don't need to spend a lot of time figuring that out now.